code/modules/clothing/under/costume.dm

```diff
@@ -145,6 +145,14 @@
 	body_parts_covered = CHEST|GROIN|ARMS
 	can_adjust = FALSE
 
+/obj/item/clothing/under/costume/kamishimo
```

Comment:
most of the curator costumes have some level of armor.

Reply:
no they dont

Reply:
Look at the jacket for the tomb raider costume.  The jumpsuits arent armores, being undersuits

Reply:
wait, im stupid, this is an undersuit... for some reason